package.json

```diff
@@ -4,10 +4,15 @@
   "scripts": {
     "dev": "ts-node src/index.ts",
     "build": "tsc",
-    "start": "node dist/index.js"
+    "start": "node dist/index.js",
+    "devDeployCommands": "ts-node src/deploy-commands.js",
+    "deployCommands": "node dist/deploy-commands.js"
```

Comment:
I think the kebab-case convention is more nice than camelCase? it's totally fine, will solve that myself

Reply:
I don't think it matters what we use as long as it's consistent.